docs/source/index.rst

```diff
@@ -11,18 +11,20 @@ Welcome to Hyperledger Indy Node's documentation!
    :maxdepth: 1
 
    transactions.md
-   requests.md
+   requests-new.md
    auth_rules.md
    pool-upgrade.md
    start-nodes.md 
    add-node.md
    helper-scripts.md
    setup-iptables.md
    node-monitoring-tools-for-stewards.md
+   node-state-diagnostic-tools.md
    ci-cd.md
    indy-file-structure-guideline.md
    write-code-guideline.md
    setup-dev.md
+   1.3_to_1.4_migration_guide.md
```

Comment:
The current Sovrin version is 1.9, so I don't think that we need to include this migration guide into index. It has no value for uses now.